### PR TITLE
Support specifying multiple docker-compose.yml files

### PR DESCRIPTION
**Pull request by kezhenxu94**
With this patch, users are able to separate some common used services into a reusable `docker-compose.yml` and specify multiple `docker-compose.yml`s in the `DockerCompose` class, like 

```python
        DockerCompose(filepath=os.path.join(docker_dir, 'http'), compose_file_name=[
            'skywalking-python-sdk/tests/plugin/docker/base-compose.yml',
            'skywalking-python-sdk/tests/plugin/http/docker-compose.yml',
        ])
        cls.compose.start()
```

### CODE REVIEW

**Comment by kezhenxu94** [comment]
BTW, we're using `testcontainers-python` in our SkyWalking Python agent, https://github.com/apache/skywalking-python/pull/10

**Comment by kezhenxu94** [comment]
@romank0 added

**Comment by Ragora** [comment]
I'll close my #91 then, I suppose I forgot to do something to bring attention to it.

**Comment by kezhenxu94** [comment]
The tests all passed, but failed to upload the coverage data to S3, I suppose it's not related to the changes here

**Comment by romank0** [comment]
@kezhenxu94 I believe tests [did](https://travis-ci.org/github/testcontainers/testcontainers-python/jobs/697591986#L699) fail for the reason:

```
=================================== FAILURES ===================================
____________________ test_can_parse_multiple_compose_files _____________________
tests/test_docker_compose.py:42: in test_can_parse_multiple_compose_files
    assert port == "3306"
E   AssertionError: assert '0.0.0.0' == '3306'
E     - 0.0.0.0
E     + 3306
```

**Comment by kezhenxu94** [comment]
@romank0 yes thanks for pointing out, it's a typo and is fixed now

**Comment by kezhenxu94** [comment]
@romank0 tests all passed

**Comment by romank0** [comment]
@kezhenxu94 good to know. Unfortunately I'm not a maintainer and I'm also waiting for quite some time for the PR I've submitted to be merged.

**Comment by kezhenxu94** [comment]
> @kezhenxu94 good to know. Unfortunately I'm not a maintainer and I'm also waiting for quite some time for the PR I've submitted to be merged.

Noticed, seems this project is not actively maintained, I plan to fork, maintain and release my own version

File: testcontainers/compose.py [hunk 1]
```diff
@@ -57,7 +59,8 @@ def __init__(
             compose_file_name="docker-compose.yml",
             pull=False):
         self.filepath = filepath
-        self.compose_file_name = compose_file_name
+        self.compose_file_names =\
+            compose_file_name if type(compose_file_name) is list else [compose_file_name]
```
Comment (tillahoffmann) [inline testcontainers/compose.py:63]:
```suggestion
            compose_file_name if isinstance(compose_file_name, (list, tuple)) else [compose_file_name]
```

**Comment by tillahoffmann** [comment]
Just a `flake8` failure at `./testcontainers/compose.py:63:101: E501 line too long (102 > 100 characters)` and then we can merge.

**Comment by kezhenxu94** [comment]
@tillahoffmann should be good now

**Pull request merged**